Makefiles/sledconf.emscripten

```diff
@@ -0,0 +1,85 @@
+# SLED configuration for emscripten/WASM builds
+# Build requirements: a recent version of the emscripten SDK (emsdk), with emcc in PATH
+
+## How to run:
+## 1. successfully build sled using this config
+## 2. get the file src/modules/out_emscripten_canvas2d.html
+## 3. put it into a directory together with sled.js and sled.wasm
+## (4. depending on your browser, pointing it at the files might not satisfy it.
+##     you will have to serve those files somewhere in that case.
+##     (static file server is enough, e.g. `python3 -m http.server --bind 127.0.0.1`))
+## 5. navigate to the HTML file in your web browser - sled should start right up!
+
+## NOTICE:
+## Use `make clean` after you modified this file
+##
+
+   ####################
+   # General settings #
+   ####################
+
+## executable name
+EM_PROJECT := sled
+
+## Debug mode
+## 0 -> Compile with optimization and without debug symbols
+## 1 -> Compile without optimization and with debug symbols and define DEBUG
+DEBUG := 0
+
+## Static vs. Dynamic linking
+## 0 -> dynamic linking (use system libs)
+## 1 -> static linking (bundle libs into exe)
+## static is the only sensible option here
+STATIC := 1
```

Comment:
Everything which is required to use the platform should be put in the platform makefile, no use setting it in the sledconf.

Applicable to `STATIC` and `DEFAULT_OUTMOD`, don't think there are any others.